### PR TITLE
Fix: Panel style fixes

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Make top-right/left border radius on first accordion panel, and bottom-right/left border radius on last accordion panel, match accordion card border radius
- Make panel headings bold and panel content medium font weight

__Note:__ I removed the checkbox style fix (making the label text align at the left when it spans multiple lines) because it was already working for checkboxes in forms. Instead I've added a custom fix for the list filter checkboxes in direct-web.

#### Trello
https://trello.com/c/AyffvRrf/5491-adslot-ui-panel-and-checkbox-list-issues-2h

#### Screenshots
screen shot 2016-07-08 at 9 51 51 am

### CODE REVIEW

File: src/styles/adslotUi/Panel.scss [hunk 1]
```diff
@@ -54,3 +60,19 @@
     }
   }
 }
+
+.card-component-content {
+  $border-radius: 5px;
```
Comment (OndrejCholeva) [inline src/styles/adslotUi/Panel.scss]: I wanted to suggest this https://github.com/Adslot/adslot-ui/blob/6647462c8092345996cf82eea0f897b7c4e9bed1/src/styles/variable.scss#L55, but looks like someone took one-size-fits-all literally.

**Comment by pphminions** [comment]
LGTM ☑️

File: src/styles/adslotUi/Panel.scss (hunk 1 above)
Comment (OndrejCholeva) [inline src/styles/adslotUi/Panel.scss]: anyway Card component is using hardcoded 5px as well, it is ok like that.

File: src/components/Main.js [hunk 2]
```diff
@@ -411,14 +411,17 @@ class AppComponent extends React.Component {
...
         <h1>Checkboxes</h1>
-        <Checkbox label="Unchecked" />
-        <br />
-        <Checkbox label="Checked" checked />
-        <br />
-        <Checkbox label="Disabled" disabled />
-        <br />
-        <Checkbox label="Checked and Disabled" checked disabled />
-
+        <div style={{ width: '200px' }}>
```
Comment (vinteo) [inline src/components/Main.js]: you can put the style into `App.scss`

File: src/styles/bootstrapOverrides/Checkbox.scss [hunk 3]
```diff
@@ -24,9 +24,5 @@ input {
 
   label {
     font-weight: $font-weight-medium;
-
-    .icheckbox {
-      margin-left: -20px; // align with bootstrap, which expects the checkbox element to be -20px
-    }
```
Comment (omgaz) [inline src/styles/bootstrapOverrides/Checkbox.scss]: I'd be very careful removing this line, I have a feeling it's direct-web/bootstrap load bearing :S Just make sure in this example and in the user list pickers user list picker you can still click the checkbox ok

Comment (patrick-ivers) [inline src/styles/bootstrapOverrides/Checkbox.scss]: You're right. It is breaking things...
screen shot 2016-07-08 at 4 35 20 pm

**Comment by vinteo** [comment]
LGTM

**Comment by omgaz** [comment]
LGTM Merging 🍥 🍜

**Pull request merged**